### ISSUE TITLE
Nitrogen in zooxanthellae parameter for eReefs project (1 term)

### ISSUE THREAD

**Issue by JordanAthertonNOC** [nvs-vocabs/P01#121]
Term to be created as:

- Biomass as nitrogen of zooxanthellae [Subgroup: in coral tissue] per unit area of the bed by model prediction

One constituent S25 term to be created for 'zooxanthellae':
https://github.com/nvs-vocabs/S25/issues/18

One constituent S13 term to be created for 'in coral tissue':
https://github.com/nvs-vocabs/S13/issues/2

**Comment by gwemon** [comment]
@JordanAthertonNOC  On reflection, I wonder here if we should not be more specific for the zooxanthellae because this is referring to zooxanthellae biomass whiile in symbiosis with the coral and not as free floating or otherwise. Maybe we should combined it with sub-group (S13 vocab)='in coral tissue'?

**Comment by gwemon** [comment]
Almost @JordanAthertonNOC  It should combine as: 
"Biomass as nitrogen of zooxanthellae [Subgroup: in coral tissue] per unit area of the bed by model prediction"

**Comment by JordanAthertonNOC** [comment]
One new P01 code created:

[Biomass as nitrogen of zooxanthellae [Subgroup: in coral tissue] per unit area of the bed by model prediction](http://vocab.nerc.ac.uk/collection/P01/current/NBZOXAM1)

**Comment by dr-shorthair** [comment]
(I assume this is in the background database - not yet visible publicly)

**Comment by gwemon** [comment]
@dr-shorthair Yes, it looks like @JordanAthertonNOC closed the ticket too early. All being well the new term will be public after 07:00BST tomorrow AM.

**Comment by JordanAthertonNOC** [comment]
The term is now live on the NVS and can be found here:
https://vocab.nerc.ac.uk/collection/P01/current/NBZOXAM1/
